<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureWindow</class>
 <widget class="QMainWindow" name="PictureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>2</width>
    <height>2</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyPaint</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>

    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Толщина</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="verticalSlider">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 168, 125);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>

     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Прозрачность</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="verticalSlider_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>0</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
       <property name="sliderPosition">
        <number>255</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>

     </item>
     <item row="1" column="3" colspan="7" rowspan="7">
      <widget class="QLabel" name="label_3"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
